generate.py

```diff
@@ -0,0 +1,593 @@
+import os
+import base64
+import olxcleaner
+import pkg_resources
+import shutil
+import tarfile
+import logging
+import tqdm
+import json
+
+from path import Path as path
+from olxcleaner.exceptions import ErrorLevel
+
+from django.conf import settings
+from django.core.exceptions import SuspiciousOperation
+from django.contrib.auth import get_user_model
+from django.db import IntegrityError
+
+from organizations.models import Organization
+
+from openedx.core.lib.extract_tar import safetar_extractall
+
+from cms.djangoapps.contentstore.views.course import create_new_course_in_store
+from cms.djangoapps.contentstore import errors as UserErrors
+
+from xmodule.modulestore import ModuleStoreEnum, COURSE_ROOT
+from xmodule.modulestore.django import modulestore
+from xmodule.modulestore.xml_importer import (
+    CourseImportException, import_course_from_xml
+)
+from xmodule.modulestore.exceptions import (
+    DuplicateCourseError, InvalidProctoringProvider
+)
+
+from xmodule.contentstore.django import contentstore
+
+from openedx_tagging.core.tagging.models import Tag, Taxonomy
+
+from openedx.core.djangoapps.content_tagging.api import (
+    create_taxonomy, get_taxonomies_for_org,
+    set_taxonomy_orgs, tag_content_object, get_content_tags,
+    resync_object_tags
+)
```

Comment:
Hmm, I guess I forgot to point out that we already have an import-export API for taxonomies that can import JSON files? 😬  https://github.com/openedx/openedx-learning/tree/main/openedx_tagging/core/tagging/import_export

Not a big deal, but it would have been nice to re-use the existing import code and JSON file format. No need to change now though, we need to wrap this up.

Reply:
Oh my bad 😅  didn't know we had that. I can address that in a future ticket for sure.